src/forecaster/forecaster.py

```
@@ -40,6 +40,12 @@ def forecast(course_enrolment: dict, program_enrolment: dict, schedule: dict, fo
         schedule  = json.load(schedule_file)
         schedule_file.close()        
 
+    # Validate inputs
+    valid, error = validate_inputs(course_enrolment, program_enrolment, schedule)
+    if not valid:
+        print(error)
```

Maybe throw an error? Idk